### ISSUE TITLE
The video shot on the vertical screen was imported into the Olive horizontal screen. How to solve it?

### ISSUE THREAD

**Issue by Eiffi** [olive-editor/olive#1063]
Using a mobile phone to shoot video, vertical shooting of the vertical screen, Olive imported into the horizontal screen. What should we do?

**Comment by Eiffi** [comment]
![111](https://user-images.githubusercontent.com/48773068/63212212-7449d780-c133-11e9-896a-2cf66f5e042c.png)
![222](https://user-images.githubusercontent.com/48773068/63212213-74e26e00-c133-11e9-88d2-405ff5f30550.png)

**Comment by NathanSalapat** [comment]
Rotate the footage to point upright, and change the sequence resolution to match the footage. You'll also have to re-center the footage but it should work.
![Olive](https://user-images.githubusercontent.com/8930456/63214473-3bcbde80-c0de-11e9-9b56-f0378886438e.png)

**Comment by Simran-B** [comment]
I guess the video is stored in the landscape orientation, but played back in portrait orientation because of some metadata in the file that instructs players to play is rotated 90° CW. Olive might ignore this metadata.

**Comment by Eiffi** [comment]
> Rotate the footage to point upright, and change the sequence resolution to match the footage. You'll also have to re-center the footage but it should work.
> ![Olive](https://user-images.githubusercontent.com/8930456/63214473-3bcbde80-c0de-11e9-9b56-f0378886438e.png)

Where can I find the edit window? Why can't I find this edit setting option in Olive?
![333](https://user-images.githubusercontent.com/48773068/63217955-32944d80-c182-11e9-84b0-d0b48d12b20a.png)

**Comment by Eiffi** [comment]
> I guess the video is stored in the landscape orientation, but played back in portrait orientation because of some metadata in the file that instructs players to play is rotated 90° CW. Olive might ignore this metadata.

Relying on the phone's gravity sensing system during video shooting. The recorded source file is also played in the computer at a normal angle. It just changed the angle of the import into Olive and could not find how to set it. Tried to use a rotation of 90 ° to solve, like the image processing program, if the layer is rotated, how does the canvas background rotate? I did not find the rotation settings for the canvas background. Did I use the Olive version of the problem?
